web-ui/src/components/menu/Menu.jsx

```diff
@@ -380,6 +380,15 @@ function Menu() {
             </ListItem>
             <Collapse in={adminOpen} timeout="auto" unmountOnExit>
               {createListJsx(adminLinks, true)}
+              {isAdmin && (
```

Comment:
This code was moved here from below.
I had to customize the styling of the button to match the look of the links in the ADMIN menu.

Reply:
Any reason not to just make it a `<ListItem>` like the others?

Reply:
I didn't realize I could attach an `onClick` to a `ListItem`. I pushed that change.